kafka-connect/build.gradle

```diff
@@ -242,3 +243,19 @@ project(':iceberg-kafka-connect:iceberg-kafka-connect-runtime') {
 
   assemble.dependsOn distZip, hiveDistZip
 }
+
+project(':iceberg-kafka-connect:iceberg-kafka-connect-transforms') {
```

Comment:
Question, what is the recommended way to keep the commit history or previous committers? using co author feature?

as is git history and previous commiters are not copied over.

Reply:
It would be great to keep the history. [Here](https://stackoverflow.com/questions/1365541/how-to-move-some-files-from-one-git-repo-to-another-not-a-clone-preserving-hi) is one way you can do it. I think you'd need to start over and re-apply the package name change, but I think it might be worth it.

Reply:
@bryanck updated it could you please check it now?

Reply:
It looks good! We'll want to merge this without squashing.

Reply:
ok squashed last commits together to have clean history. hope its ok.

Reply:
Yes LGTM